testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -344,31 +352,108 @@ async fn wait_genesis_job(kube_client: &K8sClient, era: usize) -> Result<(), any
     .await
 }
 
-pub fn clean_k8s_cluster(
-    helm_repo: String,
-    base_num_validators: usize,
-) -> Result<(), anyhow::Error> {
-    assert!(base_num_validators <= MAX_NUM_VALIDATORS);
+fn remove_validator(validator_name: &str) -> Result<()> {
+    let validator_uninstall_args = ["uninstall", "--keep-history", validator_name];
+    println!("{:?}", validator_uninstall_args);
```

Comment:
I think maybe we can change all these logs for operation args to `debug!` instead of println